docs/OV_Runtime_UG/multi_device.md

```diff
@@ -19,7 +19,7 @@ The setup of Multi-Device can be described in three major steps:
 
 1. Configure each device as usual.
 2. Load the network to the Multi-Device plugin created on top of a (prioritized) list of the configured devices. This is the only change needed in the application.
-3. As with any other ExecutableNetwork call (resulting from `InferenceEngine::Core::LoadNetwork`), you create as many requests as needed to saturate the devices.
+3. As with any other ExecutableNetwork call (resulting from `InferenceEngine::ov::compile_model`), you create as many requests as needed to saturate the devices.
```

Comment:
Replace:
1. ExecutableNetwork => CompiledModel in all places
2. InferenceEngine::ov::compile_model => ov::Compile_model

Reply:
Replace also in this document:
- executable network => compiled model

Reply:
fixed

Reply:
I still see usage of `executable network`